src/blockKit.js

```diff
@@ -81,22 +81,31 @@ module.exports.getInternalErrorMessage = function() {
 };
 
 // https://api.slack.com/tools/block-kit-builder?mode=message&blocks=%5B%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22Trivia%20question%20for%20November%208%2C%202019%5Cn%3E%20Harald%2C%20nicknamed%20Tangle-hair%2C%20took%20an%20oath%20not%20to%20cut%20or%20comb%20his%20hair%20until%20he%20became%20the%20first%20king%20of%20a%20united%20Norway.%22%7D%7D%2C%7B%22block_id%22%3A%22vote_block%22%2C%22type%22%3A%22actions%22%2C%22elements%22%3A%5B%7B%22action_id%22%3A%22vote_fact_action%22%2C%22type%22%3A%22button%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22text%22%3A%22Fact%22%7D%2C%22style%22%3A%22primary%22%2C%22value%22%3A%22fact%22%7D%2C%7B%22action_id%22%3A%22vote_crap_action%22%2C%22type%22%3A%22button%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22text%22%3A%22Crap%22%7D%2C%22style%22%3A%22danger%22%2C%22value%22%3A%22crap%22%7D%5D%7D%2C%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22%3A%201%3A%20Alice%5Cn%3A-1%3A%20Bob%5Cn%3A%201%3A%20Carolyn%5Cn%3A%201%3A%20Dave%22%7D%7D%5D
-module.exports.getOpenMessage = function({date, questionText, responses}) {
+module.exports.getOpenMessage = function({date, questionText, responses, answer, answerText}) {
```

Comment:
Let's rename this. Now that you merged `getCloseMessage` into this function its name doesn't make as much sense. Perhaps `getQuestionMessage` or `getTriviaMessage`. I'd be open to other suggestions.